files/benchmark_job.yml

```diff
@@ -39,6 +39,10 @@
                          allowed-multiselect: true
                          ignore-offline-nodes: true
                          description: 'Node with which to run the benchmark on the machine'
+                 - string:
+                         name: scripts_branch
```

Comment:
Now that we have two branches, we need to identify both for what they are. I suggest "scripts_branch" and "harness_branch".